### PR TITLE
fix(TKC-1642): update TestWorkflow Toolkit to use libssl3

### PR DESCRIPTION
**Pull request by rangoo94**
## Pull request description 

`libssl1.1` is not available in our current Alpine - https://github.com/kubeshop/testkube/actions/runs/8201841833/job/22431395514

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/tcl/testworkflow-toolkit/commands/execute.go [hunk 1]
```diff
@@ -27,19 +27,19 @@ import (
 	"github.com/kubeshop/testkube/pkg/ui"
 )
 
-type testExecutionDetails struct {
+type TestExecutionDetails struct {
```
[the remark this answers is not in the document]
Comment (rangoo94) [reply cmd/tcl/testworkflow-toolkit/commands/execute.go:30]: Actually right, I could just delete it 🙂

File: pkg/tcl/testworkflowstcl/testworkflowcontroller/logs.go [hunk 2]
```diff
@@ -39,9 +39,13 @@ func (i *Instruction) ToInternal() *testkube.TestWorkflowOutput {
 	if i == nil {
 		return nil
 	}
-	value := &i.Value
-	if i.Value == nil {
-		value = nil
+	value := map[string]interface{}(nil)
+	if i.Value != nil {
+		v, _ := json.Marshal(i.Value)
+		_ = json.Unmarshal(v, &value)
```
Comment (vLia) [inline pkg/tcl/testworkflowstcl/testworkflowcontroller/logs.go:45]: should we add a warning when there are errors?

**Review by vLia — APPROVED** [review]
LGTM, had a minor question

File: pkg/tcl/testworkflowstcl/testworkflowcontroller/logs.go (hunk 2 above)
Comment (rangoo94) [reply pkg/tcl/testworkflowstcl/testworkflowcontroller/logs.go:45]: We could add one, good point :+1:

**Pull request merged**